Home/Home.html

```diff
@@ -1,13 +1,171 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
```

Comment:
- [ ] [OPTIONAL] As best practice, it is semantically correct to add a descriptive title to your code because it allows search engines to determine what the content of your page is about thus improving your optimization and accessibility. For example, you could use `GamesPortal Global Summit` or anything that seems descriptive enough to summarize the content/focus of your page.